modules/core/src/main/build_jaicore.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<project name="imagen" basedir=".">
+
+    <property environment="env"/>
+    <property name="java.root"     value="${basedir}"/>
+    <property name="java1.dir"     value="${java.root}/java"/>
+    <property name="build.dir"   value="${java.root}/build"/>
+    <property name="classes.dir" value="${java.root}/build/classes"/>
+    <property name="jar.dir"     value="${java.root}/build"/>
+    
+    <target name="clean">
+        <delete dir="${build.dir}"/>
+    </target>
+
+    <target name="compile">
+        <mkdir dir="${classes.dir}"/>
+        <javac srcdir="${java1.dir}" destdir="${classes.dir}">
+                <compilerarg line="-encoding ISO-8859-1"/>
+                <compilerarg line="-source 1.8"/>
+                <compilerarg line="-target 1.8"/>
+                <!--
+                <compilerarg line="-classpath ${java.root}/jlib/mlibwrapper_jai.jar:${java.root}/jlib/clibwrapper_jiio.jar:${java.root}/jlib/rt.jar" />
+                -->
+                <compilerarg line="-classpath ${java.root}/jlib/mlibwrapper_jai.jar:${java.root}/jlib/rt.jar" />
```

Comment:
The {[mlibwrapper_jai}} dependency will need to be addressed, will discuss on email list.